packages/transformer/src/IModelImporter.ts

```diff
@@ -127,6 +136,7 @@ export class IModelImporter implements Required<IModelImportOptions> {
       preserveElementIdsForFiltering: options?.preserveElementIdsForFiltering ?? false,
       simplifyElementGeometry: options?.simplifyElementGeometry ?? false,
     };
+    this._duplicateCodeValueMap = new Map<Id64String, string> ();
```

Comment:
nit: (no need to fix) initialize this map in the property definition itself on line 108